model/dns_zone/dns_zone.rb

```diff
@@ -4,10 +4,9 @@
 
 class DnsZone < Sequel::Model
   one_to_one :strand, key: :id
-  many_to_one :project
-  many_to_many :dns_servers
-  one_to_many :records, class: :DnsRecord
-  one_to_one :active_billing_record, class: :BillingRecord, key: :resource_id, &:active
+  many_to_one :project, read_only: true
+  many_to_many :dns_servers, remover: nil, clearer: nil, is_used: true
+  one_to_many :records, class: :DnsRecord, remover: nil, clearer: nil
```

Comment:
What's `remover` and `clearer`? Can you add them to the commit message? I don't have any idea.